### ISSUE TITLE
Max temps

### ISSUE THREAD

**Issue by JocPro** [markasoftware/SpecDB#54]
Is it possible to have the max temps for each processor? I know that information is not easily obtainable, but that's the main reason for me to discovering SpecDB... and that info is nowhere to be found.

**Comment by markasoftware** [comment]
I think that is mainly the same for most processors, around 90-100 degrees max is recommended by AMD. Or do you mean the max temperature that it reaches in regular use with a stock cooler?

**Comment by zcskywire2** [comment]
I'm going to guess here and say he's looking for tjmax, which is the maximum safe operating tempature of the system. Which is typically 70c>100c

**Comment by JocPro** [comment]
Yeah, TjMax; but for FX's is 61°C and for Phenom II X4s is reported as
55-62°C

That's the to that I know from the top of my head; I think that for FX and
newer CPUs is available in AMD website...
